extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -177,6 +177,10 @@ export class AmpIframe extends AMP.BaseElement {
 
     /** @private {!IntersectionObserver} */
     this.intersectionObserver_ = null;
+
+    if (!this.element.hasAttribute('frameborder')) {
+      this.element.setAttribute('frameborder', '0');
```

Comment:
I believe defaults should be reflected on `amp-iframe` so they authors can see it using dev tools. Same way setting `resizable` sets `scrolling=no` on the actual `amp-iframe` rather than iframe (couple of lines above this)